frontend/src/components/Profile/Profile.tsx

```diff
@@ -32,122 +48,155 @@ const InnerProfile: React.FC = () => {
-              InputProps={{ style: { fontSize: 20, height: '3rem' } }}
-            />
-            <label className="text-2xl font-bold text-black">Email</label>
-            <TextField
-              name="email"
-              value={formData.email}
-              onChange={handleInputChange}
-              label="Email"
-              variant="filled"
-              fullWidth
-              InputProps={{ style: { fontSize: 20, height: '3rem' } }}
-            />
-            <label className="text-2xl font-bold text-black">
-              Phone Number
-            </label>
-            <TextField
-              name="phone"
-              value={formData.phone}
-              onChange={handleInputChange}
-              label="Phone Number"
-              variant="filled"
-              fullWidth
-              InputProps={{ style: { fontSize: 20, height: '3rem' } }}
-            />
-            <label className="text-2xl font-bold text-black">Address</label>
-            <TextField
-              name="address"
-              value={formData.address}
-              onChange={handleInputChange}
-              label="Address"
-              variant="filled"
-              fullWidth
-              InputProps={{ style: { fontSize: 20, height: '3rem' } }}
-            />
-            <label className="text-2xl font-bold text-black">Company</label>
-            <TextField
-              name="company"
-              value={formData.company}
-              onChange={handleInputChange}
-              label="Company"
-              variant="filled"
-              fullWidth
-              InputProps={{ style: { fontSize: 20, height: '3rem' } }}
-            />
-            <label className="text-2xl font-bold text-black">Description</label>
-            <TextField
-              name="description"
-              value={formData.description}
-              onChange={handleInputChange}
-              label="Description"
-              variant="filled"
-              fullWidth
-              InputProps={{ style: { fontSize: 20, height: '3rem' } }}
-            />
-          </div>
         </div>
-      </div>
+        <div className="relative flex flex-col justify-center ml-12 w-2/3 space-y-2">
 
-      {/* Dialog for image upload */}
+          <label className="text-2xl font-bold text-black">Name</label>
+          <TextField
+            name="name"
+            value={formData.name}
+            onChange={handleInputChange}
+            label="Name"
+            variant="filled"
+            fullWidth
+            disabled={!isEditing}
+            InputProps={{ style: { fontSize: 20, height: '3rem' } }}
+          />
+          <label className="text-2xl font-bold text-black">Email</label>
+          <TextField
+            name="email"
+            value={formData.email}
+            label="Email"
+            variant="filled"
+            fullWidth
+            InputProps={{ style: { fontSize: 20, height: '3rem' } }}
+            disabled
```

Comment:
What's the reason for having email permanently disabled?

Reply:
Emial is not editable cuz we want the email as the logging in's account

Reply:
Then we can just remove the field and not show it, I reckon that's better (only when we're editing).

Reply:
if we remove the field, the page will be empty if not click edit. We can keep the field to show the details or as a entry for user to input.